airbyte-ci/connectors/pipelines/poetry.lock

```diff
@@ -394,6 +395,7 @@ develop = false
 [package.dependencies]
 click = "^8.1.3"
 common_utils = {path = "../common_utils", develop = true}
+cryptography = ">=42.0"
```

Comment:
This could be done separately - this one needed to be locked after https://github.com/airbytehq/airbyte/pull/36614